wordpress/composer.json

```diff
@@ -121,5 +120,12 @@
     "psr-4": {
       "CDS\\": "wp-content/plugins/cds-base/classes/"
     }
+  },
+  "config": {
+    "allow-plugins": {
+      "composer/installers": true,
+      "ffraenz/private-composer-installer": true,
+      "pestphp/pest-plugin": true
+    }
```

Comment:
what does this do?

Reply:
Will remove --- think that was from a bad update